src/pt/huntersscans/src/eu/kanade/tachiyomi/extension/pt/huntersscans/HuntersScans.kt

```diff
@@ -21,5 +26,19 @@ class HuntersScans : Madara(
 
     override val useLoadMoreRequest = LoadMoreStrategy.Always
 
-    override val useNewChapterEndpoint = true
+    override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> = Observable.just(fetchAllChapters(manga))
+
+    private tailrec fun fetchAllChapters(manga: SManga, page: Int = 1, accumulator: List<SChapter> = emptyList()): List<SChapter> {
```

Comment:
Just write a simple `while (true)` loop instead of using `tailrec`. The logic is simple enough.